src/region.js

```diff
@@ -200,30 +139,6 @@ var Region = MarionetteObject.extend({
     return true;
   },
 
-  _ensureViewIsIntact: function(view) {
-    if (!view) {
-      throw new MarionetteError({
-        name: 'ViewNotValid',
-        message: 'The view passed is undefined and therefore invalid. You must pass a view instance to show.'
-      });
-    }
-
-    if (view._isDestroyed) {
-      throw new MarionetteError({
-        name: 'ViewDestroyedError',
-        message: 'View (cid: "' + view.cid + '") has already been destroyed and cannot be used.'
-      });
-    }
-  },
-
-  // Override this method to change how the region finds the DOM
-  // element that it manages. Return a jQuery selector object scoped
-  // to a provided parent el or the document if none exists.
-  getEl: function(el) {
```

Comment:
we might want to keep this for this use-case: https://github.com/marionettejs/backbone.marionette/issues/2846

Until we can provide a better jQuery alternative